scripts/setup_environment.sh

```diff
@@ -60,3 +59,7 @@ rm cmake-3.24.2-Linux-x86_64.sh
 
 export PATH="$(pwd)/.local/bin:${PATH}"
 cmake --version
+
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
I'm sure we don't want to always have the most recent version of Rust.
I was thinking maybe we have some build image where it is all preinstalled, and we just need to update one.  But installing new Rust on each build is unpreditable and flaky, IMO.

Reply:
@valeryz yes, it has been pinned to specific version otherwise as you mentioned the process will be unpredictable and flaky.